frontend/src/components/NotificationsCentral/style.scss

```diff
@@ -0,0 +1,42 @@
+@use '../../theme/variables.scss' as vars;
+
+.#{vars.$bcgov-prefix}--row{
+  margin: 0rem;
+}
+
+.btn-viewed {
+  padding-right: 3.75rem;
+}
+
+.btn-all-notifications {
+  padding-right: 3.31rem;
+}
+
+.empty-section-container {
+  padding: 1.5rem 1rem;
+}
+
+p.subtitle-section {
+  font-size: 0.875rem;
+  font-weight: normal;
```

Comment:
Use carbon types here, so it remains on the same pattern, take a look at the PageTitle's `style.scss` (lines 1 and 8). The type style can be seen on figma.